### PR TITLE
Add odometer, bring in odometer library

### PR DESCRIPTION
**Pull request by nreddy216**

### CODE REVIEW

File: _grow/source/js/counter.js [hunk 1]
```diff
@@ -9,43 +11,26 @@
...
-
-    hoursOutput.reverse();
-    for (var a = 0; a < hoursOutput.length; a++) {
-      outputHtml += hoursOutput[a] ;
-    }
-    domTarget.innerHTML = outputHtml;
+  var odometerEl = document.getElementById('odometer');
+
+  var calculateHours = function() {
+      // get current time
```
Comment (fertolg) [inline _grow/source/js/counter.js]: SUPER NIT: one extra space in this comment

File: _grow/views/base.html [hunk 2]
```diff
@@ -62,6 +62,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.20.3/TweenMax.min.js"></script>
     <script type="text/javascript" src="//www.gstatic.com/external_hosted/slick/slick.min.js"></script>
     <script src="https://cdn.jsdelivr.net/particles.js/2.0.0/particles.min.js"></script>
+    <link rel="stylesheet" href="http://github.hubspot.com/odometer/themes/odometer-theme-minimal.css" />
```
Comment (fertolg) [inline _grow/views/base.html]: Can we move this to the top with other styles?

File: _grow/source/js/hero.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+import 'odometer';
```
Comment (fertolg) [inline _grow/source/js/hero.js]: just curious why this is imported here if it's not used in js

File: _grow/source/js/counter.js [hunk 4]
```diff
@@ -9,43 +11,26 @@
   var developerSecondsSavedPerSecond = 560;
```
Comment (GeerSwitch) [inline _grow/source/js/counter.js]: Nitpick: Use ES6 let/const declarations.

File: _grow/source/js/hero.js (hunk 3 above)
Comment (nreddy216) [reply _grow/source/js/hero.js]: Oops - I meant to import it within counter. My bad.

**Pull request merged**